src/lamp_py/bus_performance_manager/write_events.py

```diff
@@ -36,6 +41,12 @@ def write_bus_metrics() -> None:
             day_logger.log_failure(FileNotFoundError(f"No RT_VEHICLE_POSITION files found for {service_date}"))
             continue
 
+        # need gtfs_rt files to run process - this happens regularly at ~12am, so this
+        # is an expected case.
+        if len(tm_files) == 0:
```

Comment:
This process will work with `tm_files == 0` I don't think this is protecting anything.